### PR TITLE
Add metadata translation keys

### PR DESCRIPTION
**Pull request by nickskyline**

### CODE REVIEW

**Review by BGMP — CHANGES_REQUESTED** [review]
Nice job! It looks good as a starting point.

I realised we may have a singular/plural issue with car classes in spanish. For example, the page title "Re-Volt America - Rookie Cars" translates to "Re-Volt America - Autos Novato"; or even worse, "Re-Volt America - Advanced Cars" to "Re-Volt America - Autos Avanzado". 

The capitalisation of the strings is also being assumed incorrectly by passing the classes as variables. See the fragment below:

https://github.com/Re-Volt-America/RVA/pull/77/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R368-R371

I'd say just create a string for each car class by separate, under the same key you've added this one, and avoid passing car classes as variables to prevent all of the issues listed above.

File: config/locales/en.yml [hunk 1]
```diff
@@ -638,7 +749,7 @@ en:
       admin: "Admin"
     badges:
       admin: "Administrator"
-      mod: "Moderator"
+      mod: "Mod"
```
Comment (BGMP) [inline config/locales/en.yml:752]: Should be "Moderator", not "Mod"

```suggestion
      mod: "Moderator"
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -488,7 +569,7 @@ en:
           success: "Session was successfully imported."
 
     results:
-      no-team-results: "No team results to display"
+      no-team-results: "No team results to disiplay"
```
Comment (BGMP) [inline config/locales/en.yml:572]: Typo
```suggestion
      no-team-results: "No team results to display"
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -176,11 +181,7 @@ en:
...
         android:
           li-1-html: "Download and install \"<b>Termux</b>\", if you can't find it at the Play Store, then use this <a href='https://f-droid.org/en/packages/com.termux/', target='_blank', rel='noopener noreferrer'>link (F-Droid)</a> to download it."
-          li-2-html: | 
-            Copy and paste the following command inside the app:<br/><br/>
-            <code>am start -n com.rvgl.rvgl/.RVGLActivity -e args "-lobby IP -pass password"</code><br/><br/>
-            Replace the IP and password accordingly before pressing Enter, as shown in the screenshot; 
-            if the game doesn't open up immediately, close the app from the notifications and try again, you may have copied something wrong.
+          li-2-html: "Copy and paste inside the app this command; <code> am start -n com.rvgl.rvgl/.RVGLActivity -e args \"-lobby &lt;ip&gt  -pass &lt;password&gt </code> you ought write the IP and password inside the text before pressing Enter as shown in the screenshot; if the game doesn't open up inmediatly, close the app from the notifications and try again, you may have copied something wrong."
```
Comment (BGMP) [inline config/locales/en.yml:184]: This was reverted for some reason? Why?

File: config/locales/en.yml [hunk 4]
```diff
@@ -52,7 +52,6 @@ en:
       logs: "Development Logs"
       bugs: "Report a Bug"
       sponsor: "Sponsor"
-      faq: "Frequently Asked Questions"
```
Comment (BGMP) [inline config/locales/en.yml:55]: Removed this for no reason?

**Review by BGMP — CHANGES_REQUESTED** [review]
I think something went wrong whilst you were rebasing your progress. Make sure to keep the changes which had been made during the last several days.

Some typos were introduced back because of this bad rebase :(.

My advice would be to revert, rebase on top of master and go with the middle option between keeping your changes only and theirs only; that option will let you cherry-pick what you want to accept and what you want to keep from your changes alone.

**Comment by nickskyline** [comment]
Apologies, it was a local file issue :(

It should be fine now.

**Comment by BGMP** [comment]
Thanks for taking care of it. I think it should be good to go now !

**Pull request merged**